### DISCUSSION TITLE
Does Relation Pages are the same as Relation Managers?

### DISCUSSION THREAD

**Discussion by hazwan91** [filamentphp/filament#14963] [Help]
### Package

Panel builder

### Package Version

v3.2

### How can we help you?

I want to use Relation Pages instead of Relation Managers but same module for managing relationship.

[Customizing the relation manager title](https://filamentphp.com/docs/3.x/panels/resources/relation-managers#customizing-the-relation-manager-title)
Based on the documentation, we can set dynamic title. So i tried using the method, even though the function doesn't exist in ManageRelatedRecords. Unfortunately its not working.

[Relation pages](https://filamentphp.com/docs/3.x/panels/resources/relation-managers#relation-pages)
Based on this part, it says that `Now, you can customize the page in exactly the same way as a relation manager, with the same table() and form().`

The way i understand is that, what relation manager have can be used in relation page.  But how can simple getTitle not working?
So i am confused at a moment. 

Is it true some of the function of relation manager does not exist in relation page? or am i doing something wrong?

btw, i try to implement something like this.
```
public static function getTitle(Model $ownerRecord, $pageClass): string {
        return '('.$ownerRecord->name.') District';
        // or $this->ownerRecord->name . 'District';
}
```

**Comment by rizalord** [comment]
For me, I can access the owner record using the getter `$this->getOwnerRecord()`.
